lineage/Analyze.py

```diff
@@ -15,13 +15,41 @@ def submit(self, fn, *args, **kwargs):
 
 
 def Analyze(X, num_states, **kwargs):
-    """ Runs a tHMM and outputs the tHMM object, state assignments, and likelihood. """
+    """ 
+    Runs a tHMM and outputs the tHMM object, state assignments, and likelihood 
+    after fitting experimental data from :func: `Analyze_list`. 
+    
+    :param X: list of objects (cells) in a lineage
+    :type X: list
+    :param num_states: the number of states to identify
+    :type num_states: int
```

Comment:
I'd rather use [Python type hints](https://docs.python.org/3/library/typing.html) which we can [import into Sphinx](https://github.com/agronholm/sphinx-autodoc-typehints). It's cleaner and shorter.

Reply:
Thank you, this is certainly more concise. 
@jclagarde could you edit based on this? I think we only keep the :param ...: and remove the :type ...: and include the type inside the () of function arguments.
Please refer to the example in the above website to see how to write the type of output (def(): ->...)
I will do this in my branch, too.

Reply:
Hello, I'm sorry for the late response -- I was in Big Bear this weekend for a club retreat. But yes, I could take a look at this and make the changes based on these suggestions. Thank you!